### ISSUE TITLE
Read songs from file - crashes when filename contains diacritics

### ISSUE THREAD

**Issue by mariokrajmer** [ChordPro/chordpro#134]
**Describe the bug**
Reading songs from file - crashes when filename contains diacritics. The message from chordpro is: "No such file or directory"

**To Reproduce**
1. create filelist with name "filelist_testing.txt" and content "..\test\názov piesňe.chordpro"
1. execute from cmd chordpro.exe -o output.pdf --filelist filelist_testing.txt
4. See error:
..\test\n├ízov pies┼łe.chordpro: No such file or directory
 at App/Music/ChordPro/Songbook.pm line 62.

**Expected behavior**
I would expect it to recognize that the described file exists and the song pdf gets generated

**Please complete the following information:**
 - OS: Windows 10
 - ChordPro version 0.974
 - did get ChordPro from SourceForge

**Additional context**
all the files i am using are in UTF-8 encoding

**Comment by ChordPro** [comment]
Confirmed.

**Comment by mariokrajmer** [comment]
I have also tested this with ChordPro version 0.977 downloaded from releases and it throws an error on different line /ChordPro.ORG/ChordPro/script/../lib/App/Music/ChordPro/Songbook.pm line 71.

**Comment by ChordPro** [comment]
Yes. The problem is not in ChordPro, but in one of the supporting modules.
I've added an updated installer kit to the release: https://github.com/ChordPro/chordpro/releases/download/R0_977/ChordPro-installer-0-977-0-2-msw-x64.exe
This should fix it.

**Comment by mariokrajmer** [comment]
The issue is now solved.
